src/components/Character.vue

```diff
@@ -1,53 +0,0 @@
-<template>
-    <div class="row">
-          <div v-for="item in items" class="card text-white bg-dark col-6 col-md-3 p-sm-1" :key="item.name">
```

Comment:
I would recommend BEM syntax since it is one of the best approaches- http://getbem.com/introduction/

Reply:
I have used bootstrap default classes. Can use BEM only in my custom classes.

Reply:
yes, I agree. The comment was for custom classes :)